OrbitGl/App.cpp

```diff
@@ -1027,49 +997,58 @@ void OrbitApp::LoadModules() {
   if (m_ModulesToLoad.size() > 0) {
     if (Capture::IsRemote()) {
       LoadRemoteModules();
-    } else {
+      return;
+    }
+#ifdef _WIN32
+    std::shared_ptr<Module> module = m_ModulesToLoad.front();
+    m_ModulesToLoad.pop();
+    GLoadPdbAsync(module);
+#else
+    while (!m_ModulesToLoad.empty()) {
       std::shared_ptr<Module> module = m_ModulesToLoad.front();
       m_ModulesToLoad.pop();
-      GLoadPdbAsync(module);
-    }
-  }
-}
 
-//-----------------------------------------------------------------------------
-bool OrbitApp::LoadRemoteModuleLocally(
-    std::shared_ptr<struct Module>& a_Module) {
-  std::string debugName = a_Module->m_Name + ".debug";
-  for (const auto& dir : m_SymbolLocations) {
-    // TODO: check that build-id in debug file
-    //       matches build-id in executable.
-    std::string fileName = dir + debugName;
-    if (Path::FileExists(fileName)) {
-      a_Module->m_PdbName = fileName;
-      PRINT_VAR(fileName);
-      GLoadPdbAsync(a_Module);
-      return true;
+      if (SymbolHelper::LoadSymbolsIncludedInBinary(module)) continue;
+      if (SymbolHelper::LoadSymbolsVisualizer(module)) continue;
+
+      ERROR("Could not load symbols for module %s", module->m_Name.c_str());
     }
+    GOrbitApp->FireRefreshCallbacks();
+#endif
   }
-  return false;
 }
 
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadRemoteModules() {
-  std::vector<std::string> modules;
+  std::vector<ModuleDebugInfo> module_infos;
+
   while (!m_ModulesToLoad.empty()) {
-    auto module = m_ModulesToLoad.front();
+    std::shared_ptr<Module> module = m_ModulesToLoad.front();
```

Comment:
std::move would be beneficial here as well.

Reply:
done